GVFS/GVFS.NativeHooks.Common/common.posix.cpp

```diff
@@ -31,22 +31,22 @@ PATH_STRING GetGVFSPipeName(const char *appName)
     }
     
     PATH_STRING finalRootPath(GetFinalPathName(enlistmentRoot));
-    size_t resultLength = strlcpy(enlistmentRoot, finalRootPath.c_str(), sizeof(enlistmentRoot));
-    if (resultLength >= sizeof(enlistmentRoot))
+    size_t enlistmentRootLength = strlen(finalRootPath.c_str());
```

Comment:
I know you're not the one to introduce this, but can you please use `strnlen` or similar, as we know (at least right now) this cannot be larger than `MAX_PATH`, and we need to move away from `strlen`

Reply:
`finalRootPath` is actually an `std::string` and so we can simply call `finalRootPath.length()` 

We could switch from `PATH_STRING` to `std::string` above on line 33, but I'm not sure for readability which is better.

Reply:
Done!